File: frontend/frontend_idea_app/lib/models/Club.dart
class Club {
  String name, imageUrl, description;
  int vote;

  Club(this.name, this.imageUrl, this.description);

  Club.fromJson(Map<String, dynamic> json)
      : name = json['name'],
        imageUrl = json['imageUrl'],
        vote = json['vote'],
        description = json['description'];

  Map<String, dynamic> toJson() => {
        "name": "'" + name + "'",
        "imageUrl": "'" + imageUrl + "'",
        "vote": vote,
        "description": "'" + description + "'",
      };
}

File: backend/ideappservice/src/main/java/com/ideapp/ideappservice/model/Club.java
package com.ideapp.ideappservice.model;

public class Club {
    String name;
    String imageUrl;
    int vote;
    String description;

    public Club(String name,String imageUrl){
        this.name = name;
        this.imageUrl = imageUrl;

    }

    public int getVote() {
        return vote;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setVote(int vote) {
        this.vote = vote;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getName() {
        return name;
    }
    public Club(){}

    public void setName(String name) {
        this.name = name;
    }
}
